src/workflows-app/SubmissionHistory.js

```diff
@@ -45,12 +47,19 @@ export const BaseSubmissionHistory = ({ namespace, workspace }, _ref) => {
           (r) => _.merge(r, { duration: getDuration(r.state, r.submission_timestamp, r.last_modified_timestamp, isRunSetInTerminalState) }),
           runSets.run_sets
         );
-        return _.merge(runSets, { run_sets: durationEnhancedRunSets });
+        // TODO: Remove filtering once WM-2232 is complete
+        const onlyWorkspaceRunSets = durationEnhancedRunSets.filter(
```

Comment:
small nit: It might be good to add a comment on why we are doing this for readability (even though it will be removed after WM-2232)